ibm/service/secretsmanager/resource_ibm_sm_configuration_private_certificate_root_CA.go

```diff
@@ -0,0 +1,724 @@
+			"max_path_length": &schema.Schema{
+				Type:        schema.TypeInt,
+				Optional:    true,
+				Computed:    true,
+				ForceNew:    true,
+				Description: "The maximum path length to encode in the generated certificate. `-1` means no limit.If the signing certificate has a maximum path length set, the path length is set to one less than that of the signing certificate. A limit of `0` means a literal path length of zero.",
+			},
+			"exclude_cn_from_sans": &schema.Schema{
+				Type:        schema.TypeBool,
+				Optional:    true,
+				ForceNew:    true,
+				Description: "Controls whether the common name is excluded from Subject Alternative Names (SANs).If the common name set to `true`, it is not included in DNS or Email SANs if they apply. This field can be useful if the common name is a human-readable identifier, instead of a hostname or an email address.",
+			},
+			"permitted_dns_domains": &schema.Schema{
+				Type:        schema.TypeList,
+				Optional:    true,
+				ForceNew:    true,
+				Description: "The allowed DNS domains or subdomains for the certificates that are to be signed and issued by this CA certificate.",
+				Elem:        &schema.Schema{Type: schema.TypeString},
+			},
+			"ou": &schema.Schema{
+				Type:        schema.TypeList,
+				Optional:    true,
+				ForceNew:    true,
+				Description: "The Organizational Unit (OU) values to define in the subject field of the resulting certificate.",
+				Elem:        &schema.Schema{Type: schema.TypeString},
+			},
+			"organization": &schema.Schema{
+				Type:        schema.TypeList,
+				Optional:    true,
+				ForceNew:    true,
+				Description: "The Organization (O) values to define in the subject field of the resulting certificate.",
+				Elem:        &schema.Schema{Type: schema.TypeString},
+			},
+			"country": &schema.Schema{
+				Type:        schema.TypeList,
+				Optional:    true,
+				ForceNew:    true,
+				Description: "The Country (C) values to define in the subject field of the resulting certificate.",
+				Elem:        &schema.Schema{Type: schema.TypeString},
+			},
+			"locality": &schema.Schema{
+				Type:        schema.TypeList,
+				Optional:    true,
+				ForceNew:    true,
+				Description: "The Locality (L) values to define in the subject field of the resulting certificate.",
+				Elem:        &schema.Schema{Type: schema.TypeString},
+			},
+			"province": &schema.Schema{
+				Type:        schema.TypeList,
+				Optional:    true,
+				ForceNew:    true,
+				Description: "The Province (ST) values to define in the subject field of the resulting certificate.",
+				Elem:        &schema.Schema{Type: schema.TypeString},
+			},
+			"street_address": &schema.Schema{
+				Type:        schema.TypeList,
+				Optional:    true,
+				ForceNew:    true,
+				Description: "The street address values to define in the subject field of the resulting certificate.",
+				Elem:        &schema.Schema{Type: schema.TypeString},
+			},
+			"postal_code": &schema.Schema{
```

Comment:
postal_code is a list type ?can we provide more than one postal code

Reply:
SM is following the pattern allowed by vault PKI - [postal_code](https://developer.hashicorp.com/vault/api-docs/v1.10.x/secret/pki#postal_code)